ibm/service/power/resource_ibm_pi_snapshot.go

```diff
@@ -35,54 +34,54 @@ func ResourceIBMPISnapshot() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
-			helpers.PISnapshotName: {
+			Arg_SnapshotName: {
 				Type:        schema.TypeString,
 				Required:    true,
 				Description: "Unique name of the snapshot",
 			},
-			helpers.PIInstanceName: {
+			Arg_InstanceName: {
 				Type:        schema.TypeString,
 				Required:    true,
 				Description: "Instance name / id of the pvm",
 			},
-			helpers.PIInstanceVolumeIds: {
+			Arg_InstanceVolumeIds: {
```

Comment:
Should be`PIInstanceVolumeIds                 = "pi_volume_ids"`
but changed to`Arg_InstanceVolumeIds                   = "pi_instance_volume_ids"`
Change value to be `pi_volume_ids`

Reply:
Changes made